pkg/syncer/syncer.go

```diff
@@ -122,3 +127,27 @@ func (s *ConfigSyncer) namespacesForSelector(selector string) (sets.String, erro
 	}
 	return ns, nil
 }
+
+func (s *ConfigSyncer) createEvent(component string, obj runtime.Object, eventType, reason, message string) (*core.Event, error) {
```

Comment:
Use event reorder to write event.
https://github.com/appscode/voyager/blob/master/pkg/eventer/recorder.go#L45

Reply:
Create the event recorder in the operator https://github.com/appscode/kubed/blob/master/pkg/operator/operator.go#L60